terraform/modules/stack/base/outputs.tf

```diff
@@ -65,11 +65,20 @@ output "rds_postgres_security_group" {
 
 /* RDS Bosh Instance */
 output "bosh_rds_url" {
```

Comment:
Again, it's not about the name of the resource itself, it's about the output..

So, we have the pattern currently
```
  output.foo_prev == resource.aws_something.foo_v1 
  output.foo_curr == resource.aws_something.foo_v2
```
Are you saying that terraform is going to destroy an instance based on the fact that the output reference changed?  That doesn't seem correct.